zerver/lib/onboarding.py

```diff
@@ -88,6 +94,13 @@ def send_initial_direct_message(user: UserProfile) -> int:
 I've kicked off some conversations to help you get started. You can find
 them in your [Inbox](/#inbox).
 """)
+        welcome_bot_custom_message_string = ""
+        # Add welcome bot custom message.
+        if welcome_message_custom_text:
+            welcome_bot_custom_message_string = _("""
+The administrators for this organization would also like to share the following information:
+```quote\n{}\n```
+""").format(welcome_message_custom_text)
```

Comment:
I would prefer to have the block-quote syntax be outside the translated string. This will save translators work if we decide to make any tweaks to the formatting later.